generators/filesystem/testing/expected.csv

```diff
@@ -0,0 +1,46 @@
+entity,attribute,value
+foo.json,id,/Users/swanickj/Code/origins-generators/generators/filesystem/testing/test_dir/foo.json
+foo.json,accessed,2015-07-08T14:48:37
+foo.json,size,0
+foo.json,uid,1288452253
+foo.json,created,2015-07-08T14:48:37
+foo.json,mode,33188
+foo.json,gid,1768498755
+foo.json,modified,2015-07-08T14:48:37
+.,file,foo.json

```

Comment:
Thinking more about this, I think the directories themselves can be ignored. Update the generator to only emit facts about files.